### PR TITLE
Enabled Zally for OpenAPI 3 specs

### PR DESCRIPTION
**Pull request by semonte**
This commit introduces Zally linting for OpenAPI 3
specifications.

### CODE REVIEW

**Review by maxim-tschumak — APPROVED** [review]
Great!

**Comment by maxim-tschumak** [comment]
:+1:

File: examples/extensions-zalando/src/main/java/org/zalando/intellij/swagger/examples/extensions/zalando/validator/zally/ZallyYamlFileValidator.java [hunk 1]
```diff
@@ -11,50 +11,33 @@
...
-import org.zalando.intellij.swagger.traversal.path.PathFinder;
 
 import java.util.List;
 import java.util.Optional;
 
-public class YamlFileValidator extends LocalInspectionTool {
+public abstract class ZallyYamlFileValidator extends LocalInspectionTool {
 
-    private final static Logger LOG = Logger.getInstance(YamlFileValidator.class);
+    private final static Logger LOG = Logger.getInstance(OpenApiYamlFileValidator.class);
```
Comment (ghost) [inline examples/extensions-zalando/src/main/java/org/zalando/intellij/swagger/examples/extensions/zalando/validator/zally/ZallyYamlFileValidator.java]: Do you really want to log using `OpenApiYamlFileValidator` (a child class of this abstract class)?

Comment (semonte) [reply examples/extensions-zalando/src/main/java/org/zalando/intellij/swagger/examples/extensions/zalando/validator/zally/ZallyYamlFileValidator.java]: Certainly not, fixed.

**Pull request merged**